### PR TITLE
Remove setResultUrl method

### PR DESCRIPTION
**Pull request by AyushBherwani1998**

### CODE REVIEW

**Review by grvgoel81 — APPROVED** [review]
lgtm

File: lib/web3auth_flutter.dart [hunk 1]
```diff
@@ -138,7 +111,11 @@ class Web3AuthFlutter {
     }
   }
 
-  static Future<void> launchWalletServices(LoginParams loginParams, ChainConfig chainConfig, {String path = "wallet"}) async {
+  static Future<void> launchWalletServices(
+    LoginParams loginParams,
```
Comment (chaitanyapotti) [inline lib/web3auth_flutter.dart:115]: i released latest android sdk. this loginParams can be removed now

Comment (AyushBherwani1998) [reply lib/web3auth_flutter.dart]: Updated

**Review by chaitanyapotti — APPROVED** [review]
lgtm

**Pull request merged**